src/backend/distributed/citus--8.0-10--8.0-11.sql

```diff
@@ -0,0 +1,19 @@
+/* citus--8.0-10--8.0-11 */
+SET search_path = 'pg_catalog';
+
+-- Deprecated functions
+DROP FUNCTION IF EXISTS worker_foreign_file_path(text);
```

Comment:
Should we drop ` worker_hash_partition_table(bigint, integer, text, text, oid, integer);` as well? That probably requires some more code clean-up as well, so I'd be ok not to do it here. [Reference](https://github.com/citusdata/citus/pull/2075/files#diff-eb0a1157a70a245b3717f65df05279e1R210) for the change.

Reply:
Removed it